build.sbt

```diff
@@ -321,6 +321,5 @@ lazy val benchmark = (project in file("benchmark"))
   .dependsOn(marathon % "compile->compile; test->test")
   .settings(
     testOptions in Test += Tests.Argument(TestFrameworks.JUnit),
-    libraryDependencies ++= Dependencies.benchmark,
-    generatorType in Jmh := "asm"
```

Comment:
Tagging @unterstein @jeschkies of https://github.com/mesosphere/marathon/commit/94efcee727612559ca227af4adce3a458281ee2c.

When set to "asm", sbt fails with https://gist.github.com/htmldoug/cbf820af93f89c4516893eb8ba4a163a and I can't get the benchmarks to run at all. Maybe you know of a better alternative than removing this.